### ISSUE TITLE
Batch normalizzation and finetuning

### ISSUE THREAD

**Issue by bhack** [pytorch/tutorials#449]
In https://github.com/pytorch/tutorials/blob/master/beginner_source/finetuning_torchvision_models_tutorial.py need you to specify to put batchnorm in eval for freezed layers/fatures extraction?

**Comment by holly1238** [comment]
See the new finetuning torchvision tutorial: https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html
